.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,32 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/debian
+{
+    "name": "QMK CLI",
+    // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+    "image": "ghcr.io/qmk/qmk_cli",
+    "customizations": {
+        "vscode": {
+            "extensions": [
+                "amodio.toggle-excluded-files",
+                "EditorConfig.EditorConfig",
+                "xaver.clang-format",
+                "llvm-vs-code-extensions.vscode-clangd",
+                "bierner.github-markdown-preview",
+                "donjayamanne.git-extension-pack"
```

Comment:
VS Code recommended the dev/remote containers extension on first setup, for me.  Not sure if we want/need it here.
```json
				"ms-vscode-remote.remote-containers"
```